docs/webmidi-mock.md

```diff
@@ -0,0 +1,177 @@
+| `calls`            | `Object`      | A register of calls made to each function, including the arguments supplied (see above). |
+| `onstatechange`    | `Function`    | A function to call when ports change (connect, disconnect, etc.).                        |
+| `addEventListener` | `Function`    | Add a listener for events the access object receives.                                    |
+| `dispatchEvent`    | `Function`    | Dispatch an event to the access object.                                                  |
+
+#### `MIDIPort` Mock
+
+| Name           | Type       | Description                                                                              |
+|----------------|------------|------------------------------------------------------------------------------------------|
+| `id`           | `String`   | The unique ID of the device associated with the port.                                    |
+| `name`         | `String`   | The name of the device associated with the port.                                         |
+| `manufacturer` | `String`   | The manufacturer of the device associated with the port.                                 |
+| `type`         | `String`   | The type of port, either "input", or "output".                                           |
+| `state`        | `String`   | The state of the port, either "open", "pending", or "closed".                            |
+| `connected`    | `String`   | "connected" if the port is connected, "disconnected" otherwise.                          |
+| `calls`        | `Object`   | A register of calls made to each function, including the arguments supplied (see above). |
+| `open`         | `Function` | A function that will handle requests to open a connection to this port.                  |
+| `close`        | `Function` | A function that will handle requests to close the connection to this port.               |
+
+#### `MIDIInput` Mock
+
+In addition to the properties and functions supported by a `MIDIPort` (see above), a MIDI input has two additional
+methods:
+
+| Name               | Type          | Description                                   |
+|--------------------|---------------|-----------------------------------------------|
+| `addEventListener` | `Function`    | Add a listener for events the input receives. |
+| `dispatchEvent`    | `Function`    | Dispatch an event to the input.               |
+
+#### `MIDIOutput` Mock
+
+In addition to the properties and functions supported by a `MIDIPort` (see above), a MIDI output has two additional
+methods:
+
+| Name    | Type       | Description                                                                             |
+|---------|------------|-----------------------------------------------------------------------------------------|
+| `send`  | `Function` | A function that handles requests to send messages to this port.                         |
```

Comment:
Function something -> something?

Reply:
I refactored the method docs to look more like the invoker docs.